egs/fisher_swbd/s5/local/swbd1_prepare_dict.sh

```diff
@@ -20,10 +20,7 @@ srcdict=$srcdir/swb_ms98_transcriptions/sw-ms98-dict.text
 
 #(2a) Dictionary preparation:
 # Pre-processing (Upper-case, remove comments)
-awk 'BEGIN{getline}($0 !~ /^#/) {$0=tolower($0); print}' \
-  $srcdict | sort | awk '($0 !~ /^[:space:]*$/) {print}' \
-   > $dir/lexicon1.txt || exit 1;
-
+grep -v '^#' $srcdict | tr '[a-z]' '[A-Z]' | awk 'NF>0' | sort > $dir/lexicon1.txt || exit 1; 
```

Comment:
The comment said 'Upper-case' but the code was actually lower-casing.  So please lower-case instead, and change the comment.  See if there are similar issues elsewhere.